libres/lib/config/conf.cpp

```diff
@@ -1037,8 +1037,9 @@ conf_instance_validate_sub_instances(const conf_instance_type *conf_instance) {
     return ok;
 }
 
-bool conf_instance_get_path_error(const conf_instance_type *conf_instance) {
-    bool path_errors = false;
+static std::set<std::string> _internal_conf_instance_get_path_error(
```

Comment:
The real reason for the split is that the logic is applied recursively (second block in `_internal_conf_instance_get_path_error`) + I want to use `std::set` to avoid duplicates (although I'm not 100% sure if this can happen). Then, because the method is exposed to C it cannot return `std::set`  hence the split.

Perhaps a comment in the code to explain this is sufficient?

Reply:
I think some documentation would suffice yes :)